dashboard/app/jobs/evaluate_rubric_job.rb

```diff
@@ -0,0 +1,45 @@
+class EvaluateRubricJob < ApplicationJob
+  queue_as :default
+
+  rescue_from(StandardError) do |exception|
+    if rack_env?(:development)
+      puts "EvaluateRubricJob Error: #{exception.full_message}"
+    end
+    raise
+  end
+
+  # 2D Map from unit name and level name, to the name of the lesson files in S3
+  # which will be used for AI evaluation.
+  # TODO: This is a temporary solution. After the pilot, we should at least make
+  # the S3 pointer editable on levelbuilder, and eventually make all of the data
+  # it points to editable there too.
```

Comment:
one complicating factor with moving all the data into levelbuilder is that js code samples often contain answer keys, but we could work around this by encrypting that data.

Reply:
Yeah that is tricky. I think we could stick to an S3 link if that would work better. We didn't realize that examples were included in the AI prompts when building the CW experience :/

Reply:
It seems like we have a working model that just expects a bucket path, so that seems like a reasonable approach. By encrypting the data, you mean store it in the repo encrypted and decrypt on seed like some level data we have? I don't mind that either. For either, I wonder what the local dev experience feels like... we will need a couple of levels that have unencrypted data for testing for those without our secrets.

Reply:
ok, let's have the next step be to store an S3 link in levelbuilder. if either of you were thinking this should be done before pilot launch please lmk, otherwise we can plan to do it after the pilot. then we can think about moving more into levelbuilder as we get closer to wanting curriculum writers to be able to edit our ai prompts, examples, etc.

> We didn't realize that examples were included in the AI prompts when building the CW experience :/

Sorry about this Bethany! I hope we can put to use what you've built there soon.

> By encrypting the data, you mean store it in the repo encrypted and decrypt on seed like some level data we have? I don't mind that either. 

yes, the way we do this for levels is what made me think of this option, and that sounds accurate for when to decrypt.

> For either, I wonder what the local dev experience feels like... we will need a couple of levels that have unencrypted data for testing for those without our secrets.

agreed! 

as another option, i started to look into how we securely store the links to exemplars which only verified teachers can see, but I think I want to put that investigation aside until we get closer to implementing this.

Reply:
Definitely feels post-pilot to me.

And don't apologize! One of the downsides of me not engaging too much in the AI side :)

Reply:
https://codedotorg.atlassian.net/browse/AITT-247